contrib/seeds/README.md

```diff
@@ -10,9 +10,8 @@ to addrman with).
 
 The seeds compiled into the release are created from sipa's DNS seed data, like this:
 
-    curl https://bitcoin.sipa.be/seeds.txt.gz | gzip -dc > seeds_main.txt
+    curl -s http://bitcoin.sipa.be/seeds.txt.gz | gzip -dc > seeds_main.txt
```

Comment:
Why are you changing this?

Reply:
These issues have all been resolved. The 3 affected files are now up to date with master. These changes were unintended and were likely a byproduct of an earlier pull. 

There should only be 3 files that change within this pull request.

1. makeseeds.py - Instantiation of ASN Decoder Class
2. asndecode.py - IP -> ASN Class Object 
3. ip2asn.tsv.gz - Database File

As a technical note the contents of asndecode.py could be merged with makeseeds.py but this would reduce clarity IMO.